src/lib/holocene/data-encoder-settings.svelte

```diff
@@ -108,8 +108,8 @@
               id="use-configuration-endpoint-radio"
               data-testid="use-configuration-endpoint-input"
             />
-            Uses {namespaceOrCluster}-level settings, where available.
-            Otherwise, use my local setting.
+            Use {namespaceOrCluster}-level settings, where available.
+            Otherwise, use my browser setting.
```

Comment:
Is this actually how the feature works? If yes, we should remove the cases of "override" in copy.

Reply:
Removed override in copy.